beater/server_test.go

```diff
@@ -13,7 +13,10 @@ import (
 	"testing"
 	"time"
 
-	"github.com/kabukky/httpscerts"
```

Comment:
this dependency is not used anymore, should be removed then (also remember to run `make notice` after removing it to remove its entry from the NOTICE file as well)

Reply:
This dependency is already removed. You are looking in the wrong side of the diff :)

Reply:
Sorry, I meant the actual code from the dependency, not the import.
https://github.com/elastic/apm-server/blob/master/vendor/vendor.json#L1163-L1168 and https://github.com/elastic/apm-server/tree/master/vendor/github.com/kabukky/httpscerts

Reply:
Doesn't look like this has been addressed yet.

Reply:
Resolved !